src/operator/timeout.ts

```diff
@@ -97,6 +93,6 @@ class TimeoutSubscriber<T> extends Subscriber<T> {
   }
 
   notifyTimeout(): void {
-    this.error(this.errorToSend);
+    this.error(new TimeoutError());
```

Comment:
Not sure which object is unused, pr #2132 is simply changing where error object is created and pass into here. Would you elaborate bit what I'm missing?